### PR TITLE
Automatic process that calls cornflow periodically

### PR DESCRIPTION
**Pull request by marioncottard**
"Test automatic process on airflow that calls cornflow for the creation of an instance and for it to be solved periodically". Also solves the bug on the roadef dag with xmltodict 0.13.0

### CODE REVIEW

File: cornflow-dags/DAG/call_cornflow.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Full imports
```
Comment (ggsdc) [inline cornflow-dags/DAG/call_cornflow.py:1]: Ok, Can we store this script as an example on our drive, but not have it here for now?

File: cornflow-dags/DAG/pulp_example_data.json [hunk 2]
```diff
@@ -0,0 +1 @@
...value": 1}]}, {"sense": -1, "pi": null, "constant": -1, "name": "color_pair_9_6_18", "coefficients": [{"name": "node_color_(6,_9)", "value": 1}, {"name": "node_color_(18,_9)", "value": 1}]}, {"sense": -1, "pi": null, "constant": -1, "name": "color_pair_9_6_19", "coefficients": [{"name": "node_color_(6,_9)", "value": 1}, {"name": "node_color_(19,_9)", "value": 1}]}, {"sense": -1, "pi": null, "constant": -1, "name": "color_pair_9_7_9", "coefficients": [{"name": "node_color_(7,_9)", "value": 1}, {"name": "node_color_(9,_9)", "value": 1}]}, {"sense": -1, "pi": null, "constant": -1, "name": "color_pair_9_7_11", "coefficients": [{"name": "node_color_(7,_9)", "value": 1}, {"name": "node_color_(11,_9)", "value": 1}]}, {"sense": -1, "pi": null, "constant": -1, "name": "color_pair_9_7_13", "coefficients": [{"name": "node_color_(7,_9)", "value": 1}, {"name": "node_color_(13,_9)", "value": 1}]}, {"sense": -1, "pi": null, "constant": -1, "name": "color_pair_9_7_17", "coefficients": [{"name": "node_color_(7,_9)", "value": 1}, {"name": "node_color_(17,_9)", "value": 1}]}, {"sense": -1, "pi": null, "constant": -1, "name": "color_pair_9_7_18", "coefficients": [{"name": "node_color_(7,_9)", "value":
```
Comment (ggsdc) [inline cornflow-dags/DAG/pulp_example_data.json:1]: Guess this ain't needed also

**Review by ggsdc — APPROVED** [review]
Just leave thos two files out of the repo for now. I guess that you tested it and there was no problem, right?

**Comment by marioncottard** [comment]
Yes, I tested, there was no problem. Did you upload them to the drive already or should I?

**Comment by marioncottard** [comment]
Also (just out of curiosity), what do we need this dag for ?

**Comment by ggsdc** [comment]
> Also (just out of curiosity), what do we need this dag for ?

Is a use case that may be needed on some deployments that we do for someone.

**Review by ggsdc — APPROVED** [review]

**Pull request merged**